src/tests/ftest/util/test_utils_container.py

```diff
@@ -336,6 +337,11 @@ def create(self, uuid=None, con_in=None, acl_file=None):
                 cop.srv_verify = con_in[2]
                 cop.chksum_type = con_in[3]
                 cop.chunk_size = con_in[4]
+                cop.rf_lvl = con_in[5]
+                kwargs["con_prop"] = cop
+            else:
+                cop = self.input_params.get_con_create_params()
+                cop.rf_lvl = ctypes.c_uint64(DaosContPropEnum.DAOS_PROP_CO_REDUN_RANK.value)
```

Comment:
Aren't we supposed to use `DaosContPropEnum.DAOS_PROP_CO_REDUN_DEFAULT`?
Also, it would be better if you could improve the logic a bit.
```
# Refer daos_api for setting input params for DaosContainer.
cop = self.input_params.get_con_create_params()

if con_in:
    cop.type = con_in[0]
    cop.enable_chksum = con_in[1]
    cop.srv_verify = con_in[2]
    cop.chksum_type = con_in[3]
    cop.chunk_size = con_in[4]
    cop.rf_lvl = con_in[5]
else:
    cop.rf_lvl = ctypes.c_uint64(DaosContPropEnum.DAOS_PROP_CO_REDUN_RANK.value)

kwargs["con_prop"] = cop
```

Reply:
> Aren't we supposed to use `DaosContPropEnum.DAOS_PROP_CO_REDUN_DEFAULT`? Also, it would be better if you could improve the logic a bit.

For CI test, we supposed to use rank rather than daos default node to reduce number of servers requirement for testing, this is a bit tricky though.

> 
> ```
> # Refer daos_api for setting input params for DaosContainer.
> cop = self.input_params.get_con_create_params()
> 
> if con_in:
>     cop.type = con_in[0]
>     cop.enable_chksum = con_in[1]
>     cop.srv_verify = con_in[2]
>     cop.chksum_type = con_in[3]
>     cop.chunk_size = con_in[4]
>     cop.rf_lvl = con_in[5]
> else:
>     cop.rf_lvl = ctypes.c_uint64(DaosContPropEnum.DAOS_PROP_CO_REDUN_RANK.value)

Sure this is good idea.

> 
> kwargs["con_prop"] = cop
> ```